app/assets/javascripts/statistics.js

```diff
@@ -293,3 +306,8 @@ Number.prototype.format = function(n, x) {
   var re = '(\\d)(?=(\\d{' + (x || 3) + '})+' + (n > 0 ? '\\.' : '$') + ')';
   return this.toFixed(Math.max(0, ~~n)).replace(new RegExp(re, 'g'), '$1,');
 };
+
+function toggleBalanceSpinner(state) {
+  spinner = $('.balance-chart.spinner-backround');
```

Comment:
@shir before coding in mind I plan more code in this function (disable buttons and etc.), but this not useful. I remove `spinner =`.